### PR TITLE
Allow controller to ask the model for isSearch().

### PR DESCRIPTION
**Pull request by dereuromark**
Resolves https://github.com/FriendsOfCake/search/issues/84
```php
$articles = $this->paginate($query);
debug($this->Articles->isSearch());

$this->set('isSearch', $this->Articles->isSearch()); 
```

### CODE REVIEW

**Comment by dereuromark** [comment]
We could make 'isSearch' just a protected $isSearch attribute on the behavior.
No need to expose it.

A cleaner approach would be to have each finder process() method return a bool.
True indicates that you actually modified the query, false would mean you bailed early, no modification done/needed after all.
Would that be a more desired solution here?

**Comment by ADmad** [comment]
CS error.

File: src/Model/Behavior/SearchBehavior.php [hunk 1]
```diff
@@ -32,9 +32,15 @@ class SearchBehavior extends Behavior
...
             'filterParams' => 'filterParams',
-            'searchManager' => 'searchManager'
-        ]
+            'searchManager' => 'searchManager',
+            'isSearch' => 'isSearch'
+        ],
     ];
+    
+    /**
+     * @var bool
```
Comment (ADmad) [inline src/Model/Behavior/SearchBehavior.php]: Description missing

**Comment by dereuromark** [comment]
Also added docs.

**Pull request merged**